README.md

```diff
@@ -62,28 +62,96 @@ The tool is a bit interactive, and can ask several question. The dialog looks li
 
 ```
 [+] Project: https://github.com/apache/poi
-[+] Let's get info about the project and calculate a rating
+[+] Let's get info about the project and calculate a security rating
+[+] Counting how many commits have been done last three months ...
```

Comment:
Counting how many commits have been done in last three months ...

Reply:
I am not sure whether the current version is correct since I am not a native speaker. I'll update the sentences if you think they should use "in".